index.html

```diff
@@ -29,9 +30,16 @@
       </div>
 
       <div class="col-lg-6 col-md-12">

```

Comment:
I am wondering, how can I put a To Do list before a calculator? i.e. if I want the todo list on the left, it doesn't look like that will be possible